web-react-ts/.dockerignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
need a newline here, hopefully once we get a linter in here that'll be less of an issue